CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# Changelog
+
+This project isn't currently versioned, so for now each PR will act as a "version".
```

Comment:
What is the data point to decide when to version this project?

Reply:
Hmm great question. I suppose we could start versioning now and use the milestones as versions (they are already kinda laid out that way).